src/components/utils/renderTable-ExportCsv.tsx

```diff
@@ -9,11 +9,7 @@ import { Box } from '@mui/system';
 import { CsvExport } from '../spreadsheet/export-csv';
 import { CustomAGGrid } from '../custom-aggrid/custom-aggrid';
 import React, { FunctionComponent, Ref } from 'react';
-import {
-    ColDef,
-    RowDataUpdatedEvent,
-    RowStyle,
-} from 'ag-grid-community/dist/lib/main';
+import { ColDef, RowDataUpdatedEvent, RowStyle } from 'ag-grid-community';
 import { GridReadyEvent, RowClassParams } from 'ag-grid-community';
```

Comment:
There is some imports to refactors
```suggestion
import {
    ColDef,
    GridReadyEvent,
    RowClassParams,
    RowDataUpdatedEvent,
    RowStyle,
} from 'ag-grid-community';
```